server/controllers/contest.js

```diff
@@ -0,0 +1,58 @@
+const Contest = require("../models/Contest");
+const asyncHandler = require("express-async-handler");
+
+exports.createContest = asyncHandler(async (req, res) => {
+    try {
+        // expecting req.body to come back in this format.
+        // {
+        // title: req.body.title,
+        // description: req.body.description,
+        // prizeAmount: req.body.prizeAmount,
+        // deadlineDate: req.body.deadlineDate,
+        // not positive how the userid is going to come from the front-end, leaving req.body for now, will change based on how id is being sent from front to back. 
+        // userId: req.body.userId
+        // }
+        const contest = await Contest.create(req.body);
+        res.status(200).json(contest);
+    } catch (err) {
+        res.status(404).json(err);
+    }
+})
+
+exports.updateContest = asyncHandler(async (req, res) => {
+    try {
+        const update = await Contest.findOneAndUpdate({
+            _id: req.params.id
+        }, {
+            $set: {
```

Comment:
Maybe use request body (`req.body`) here to update multiple fields?

Reply:
We were discussing only allowing a contest creator to update one field at a time, and eventually have them to only be able to update certain things, going to make the change and discuss with the whole team if they would eventually like to implement something different. Thank you for all insights.